docs: get enums from model into Graphql Types (#28)
GeekHunter-Brasil/hello-world: backend-styleguide/README.md

], scope: true
```

> ### 👉 When use Enum in Graphql Types get from model

Reviewer: We should probably mention `enumerize` here.

```suggestion
### 👉 When use Enum in Graphql Types get from model

We use the [enumerize](https://github.com/brainspec/enumerize) gem to handle creating enums in our models and databases.

They should be used to specify GraphQL enums.
```

❌ Bad

```ruby
module Types
  module Enums
    module Candidate
      class SituationEnumType < Types::BaseEnum

>         value 'registered', 'When the user just register to the platform.',

Reviewer: ```suggestion
        value 'registered', 'When the user has just registered on the platform.',
```

              value: 0

>         value 'register_complete', 'When the user is registered and with his profile complete.',

Reviewer: ```suggestion
        value 'register_complete', 'When the user is registered and with his profile is complete.',
```

              value: 1

>         value 'waiting_approval', 'When the user has passed a required test and are waiting to be approved and available to the companies.',

Reviewer: ```suggestion
        value 'waiting_approval', 'When the user has passed a required test and is waiting to be approved and available to the companies.',
```

], scope: true
```

> ### 👉 When use Enum in Graphql Types get from model

Reviewer: Another typo

```suggestion
### 👉 Declaring GraphQL enums with enumerize
```
